static/js/protein/chart.ts

```diff
@@ -29,10 +29,61 @@ export function drawTissueScoreChart(
   id: string,
   data: { name: string; value: string }[]
 ): void {
-  // TODO: convert data to tmpData format.
-  const tmpData = [{ x1: 20, x2: 60, y1: 30, y2: 50 }];
+  const tmpData = [
+    { name: "Cartilage", value: 3 },
+    { name: "Eye", value: 2 },
+    { name: "Liver", value: 1 },
+    { name: "Skin", value: 0 },
+  ];
+  const testData = data;
```

Comment:
also, since this is a rather long function, could you please add some comments within for the various "chunks" of code, e.g. when drawing axis, bars, etc

also helpful to comment on what the arguments are that are passed in to the function.

Reply:
Added comments and removed the unused code.